games-arcade/opensyobonaction/opensyobonaction-2.rc2.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="An open-source cross-platform port of the Japanese platformer using SDL"
```

Comment:
As the summary is what's displayed in list-mode in HaikuDepot, it has to be descriptive and concise. I'd go with something targetted at the enduser (he doesn't care about multi-platform, license or used framework). How about:
"A jump 'n' run game similar to Super Mario, but more evil"
For the description, I'd go for less forced humour and more actual gameplay... :)